site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -620,7 +646,76 @@ components:
          description: Allows arbitrary properties as extension to the ones
                       specified by the Hono API.
 
-# Tenant schema
+
+      PagingSortingFilteringOptions:
+         type: object
+         additionalProperties: false
+         properties:
+            "page":
+               $ref: '#/components/schemas/PagingOptions'
+            "filters":
+               $ref: '#/components/schemas/FilterOptionsList'
+            "sort":
+               $ref: '#/components/schemas/SortingOptions'
+
+      PagingOptions:
+         type: object
+         additionalProperties: false
+         description: options to page the results of bulk queries.
+         properties:
+            "limit":
+               type: integer
+               default: 30
+               minimum: 0
+               maximum: 200
+               description: The maximum number of objects to include in a response.
+            "offset":
+               type: integer
+               default: 0
+               minimum: 0
+               description: |
+                  The offset to use as a starting point to return results. This allows to go through the query results in
+                  a paged manner when the query returns more results than the limit.
+
+      FilterOptionsList:
+         type: array
+         description: A list of filter expressions. Objects must match all filters in order to be included in a result.
+         items:
+            $ref: '#/components/schemas/FilterOption'
+
+      FilterOption:
+            type: object
+            additionalProperties: false
+            description: Filters to be applied to bulk queries.
+            properties:
+               "field":
+                  type: string
+                  description: The JSON Path representation of the field to use for filtering.
+               "value":
+                  type: string
+                  description: The value to filter the field with.
```

Comment:
will this be matched exactly or can we use a regex here or wildcards?

Reply:
I don't know what is the better choice honnestly. I suppose it depends on the underlying database, what does it supports. 
@dejanb @ctron any opinions on this ?

Reply:
Again, I am not arguing that this *must* support wildcards or regex. In any case, we must document the expected behavior ...